caput/mpiarray.py

```diff
@@ -1,4 +1,9 @@
-"""An array class for containing MPI distributed array.
+"""MPI-distribued numpy array.
+
+:py:mod:`~caput.mpiarray` provides a distributed array object which is a subclass of
+:py:class:`numpy.ndarray`. The goal of :py:mod:`~caput.mpiarray` is to enable seamless
```

Comment:
```suggestion
This module provides a distributed array object which is a subclass of
:py:class:`numpy.ndarray`. The goal of this module is to enable seamless
```
These are self-links which aren't helpful.